.github/workflows/client.yml

```diff
@@ -25,6 +25,10 @@ jobs:
         k8s:
           - v1.21
           - v1.26
+        tls:
+          - "openssl-tls"
+          - "rustls-tls"
+    name: "local (k8s ${{ matrix.k8s }}, ${{ matrix.tls }})"
     timeout-minutes: 15
```

Comment:
OpenSSL builds look to be quite a bit slower, so unfortunately we'll need to increase this.